boards/arm/gd32f450z_eval/gd32f450z_eval.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2022, Rtone.
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: gd32f450z_eval
+name: GigaDevice GD32F450Z-EVAL
+type: mcu
+arch: arm
+ram: 256
+flash: 3072
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - pwm
```

Comment:
this is missing a few entries (I see other boards should be fixed as well)
- gpio
- dac
- spi
- i2c

this is used by the testing infrastructure

Reply:
yes thx, i should probably add `uart` too